### ISSUE TITLE
Future updates like the PyTorch version?

### ISSUE THREAD

**Issue by francismontalbo** [qubvel/segmentation_models#556]
Really hoping that this repo would have an update like the PyTorch version :( 

Really love your work.

**Comment by kp3393** [comment]
Agreed, great and valuable work! Please, updates like the Pytorch version will be extremely helpful.
